plugins/services/src/js/reducers/serviceForm/Volumes.js

```diff
@@ -21,6 +22,17 @@ const mapLocalVolumes = function (volume) {
   };
 };
 
+const mapExternalVolumes = function (volume) {
+  if (this.docker &&
+    this.dockerType === 'DOCKER' && volume.external.size != null) {
+    return Object.assign({},
```

Comment:
Please respect the `omit` interface and remove the second `Object.assign` as it isn't needed.

`Object.assign({},  volume, {external: omit(volume.external, ['size']))});`